src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -81,12 +80,20 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
             CurrentConfiguration = this.configurationProvider.GetConfiguration();
         }
 
-        private async void OnActiveSolutionChanged(object sender, EventArgs e)
+        private async void OnActiveSolutionChanged(object sender, bool isSolutionOpen)
         {
-            await UpdateConnection();
+            // An exception here will crash VS
```

Comment:
Miscellaneous: defensive so we don't crash (which happened with some of my work-in-progress changes)